guides/doc-Managing_Hosts/topics/proc_upgrading-rhel7-hosts-to-rhel8.adoc

```diff
@@ -0,0 +1,24 @@
+[id="upgrading-hosts-from-rhel7-to-rhel8_{context}"]
+= Upgrading Hosts from RHEL 7 to RHEL 8
+
+You can use a job template to upgrade your Red{nbsp}Hat Enterprise{nbsp}Linux 7 hosts to Red{nbsp}Hat Enterprise{nbsp}Linux 8.
+
+.Prerequisites
+
+* Ensure that your RHEL 7 hosts meet the requirements for the upgrade to RHEL 8. For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/planning-an-upgrade_upgrading-from-rhel-7-to-rhel-8[Planning an upgrade] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
+* Prepare your hosts for the upgrade. For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/preparing-a-rhel-7-system-for-the-upgrade_upgrading-from-rhel-7-to-rhel-8[Preparing a RHEL 7 system for the upgrade] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
+* Enable remote execution feature on {Project}. For more information, see xref:Running_Jobs_on_Hosts[].
+* Distribute {Project} SSH keys to the hosts that you want to upgrade. For more information, see xref:sect-Managing_Hosts-Establishing_a_Secure_Connection_for_Remote_Commands[].
+
+.Procedure
+
+. In the Satellite web UI, navigate to *Hosts* > *All Hosts*.
+. Select the hosts that you want to upgrade to Red{nbsp}Hat Enterprise{nbsp}Linux 8.
+. In the upper right of the Hosts window, from the *Select Action* list, select *Preupgrade check with Leapp*.
+. Click *Submit* to start the pre-upgrade check.
+. When the check is finished, click the *Leapp preupgrade report* tab and fix all issues. Note that you can fix some issues that has remediation automatically. You need to fix other issues manually.
```

Comment:
`You need to fix other issues manually.` -> `You may need to fix other issues manually.` ? there's a big chance for some manual activity, but it's not always required I think. Also if I'm not mistaken, only inhibitors need to be fixed, otherwise, the upgrade may still work. It's just risk.

I think we should also document, how to fix issues that have remediation automatically. It should be something like you select what you want to fix, click Run remediation or something. I don't know the exact button captions.